operator/pkg/resources/statefulset_test.go

```diff
@@ -160,15 +160,7 @@ func TestEnsure(t *testing.T) {
 				return np.Name == tt.nodePoolName
 			})
 			assert.NotEqual(t, -1, npIndex, "could not find nodePool")
-			cfg, err := resources.CreateConfiguration(context.TODO(),
-				c,
-				nil,
-				tt.pandaCluster,
-				"cluster.local",
-				types.NamespacedName{},
-				types.NamespacedName{},
-				TestBrokerTLSConfigProvider{},
-			)
+			cfg, err := resources.CreateConfiguration(context.TODO(), c, nil, tt.pandaCluster, "cluster.local", types.NamespacedName{}, types.NamespacedName{}, types.NamespacedName{}, TestBrokerTLSConfigProvider{})
```

Comment:
Where did all this reformatting come from? It doesn't look like anything changed but it's kinda hard to tell.